### DISCUSSION TITLE
Is it possible to run the monai bundle model inference on CPU

### DISCUSSION THREAD

**Discussion by Bala93** [Project-MONAI/MONAI#5955] [Q&A] [answered]
I am trying to run the inference of https://github.com/Project-MONAI/model-zoo/tree/dev/models/brats_mri_segmentation. I have only 12GB GPU, and it is resulting in out-of-memory error. Is it possible to run the inference in CPU ?

**Comment by yiheng-wang-nv** [comment, marked as answer]
Hi @Bala93 , could you please change this line:
https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_segmentation/configs/inference.json#L12
and set the device to cpu?
In addition, I think you can also try to reduce the `roi_size` to small values, such as `(120, 120, 80)`

**Comment by Bala93** [comment]
Hi @yiheng-wang-nv Thanks for your answer. Fyi, I am trying to run the Brats segmentation through Slicer, like this example (https://github.com/Project-MONAI/model-zoo/tree/dev/models/pancreas_ct_dints_segmentation). I tried changing the roi like you suggested, but I was using Task01_Brain tumor (Segmentation decathlon), hence there was a conflict in dimensions. Now, I have downloaded the Brats2019, and each patient has 4 different modalities. May I know what should be the input? Is it like 4 separate files, or single file with stacked volume ? I would really appreciate your help.

**Reply by yiheng-wang-nv** [reply]
I think both methods are fine. If providing 4 separate files, as for the data list, you just need to put 4 filenames into a list for each sample, and the `LoadImaged` transform can handle it.
